java-notification/pom.xml

```diff
@@ -3,6 +3,7 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
```

Comment:
For the sake of consistency, adding this groupId element. "com.google.cloud" is the groupID of this artifact:
https://search.maven.org/artifact/com.google.cloud/google-cloud-notification